### PR TITLE
Fix typos & remove a redundant function call

### PR DESCRIPTION
**Pull request by preciz**
Title says all.
Thanks!

### CODE REVIEW

**Comment by preciz** [comment]
I have also found a function call that I believe doesn't change the return value of the function.

**Comment by preciz** [comment]
Is the second commit correct or should I remove it?

File: lib/phoenix_live_view/router.ex [hunk 1]
```diff
@@ -75,7 +75,6 @@ defmodule Phoenix.LiveView.Router do
       module
       |> Atom.to_string()
       |> String.split(".")
-      |> Enum.drop(-1)
```
Comment (chrismccord) [inline lib/phoenix_live_view/router.ex]: What's the rationale for removing this?

**Comment by chrismccord** [comment]
Perfect. Thank you! ❤️❤️❤️🐥🔥

**Pull request merged**